tiltIndicator_tiltIndicatorAfter_workflow.Rmd

```diff
@@ -0,0 +1,213 @@
+---
+title: "tiltInidicator_tiltIndicatorAfter_workflow"
+author: "Kalash"
+date: "2023-07-21"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
```

Comment:
- [ ] I'll add some more useful options. For example `comment = "#>"` allows users to copy-paste-run the code they see in the article.